docs/edu/bootcamps.md

```diff
@@ -0,0 +1,49 @@
+# Running Bootcamps
+
+We collect all bootcamp materials in the [LEAP-Pangeo bootcamp repository](https://github.com/leap-stc/LEAP-bootcamps). Please keep all relevant information and materials in this repository to make it easier for participants to find them.
+
+## Preparing for the bootcamp
+
+**Past attendees have suggested that it is very helpful to have the instructor write code from scratch.**
+This ensures the following:
+
+- The lecturer takes substantially more time to write live, ensuring that others can follow along.
+- The lecturer is not a superhuman, but also makes mistakes and has to debug code. This is a very important lesson for students to learn, as it is a very common part of the coding process and often not shown in lectures.
+- It also allows the lecturer to explain the code as they write it, which is a very important part of the learning process.
+
+If you are going to use a notebook pre-populated with code, please clear all outputs prior to starting the lecture by right clicking on the notebook and selecting "Clear All Outputs".
```

Comment:
Your call, but it might be helpful for some to add a few more steps. Something like, *... in the jupyter-notebook click.... top taskbar, kernel, clear all outputs, save etc..)*